lib/matplotlib/text.py

```diff
@@ -1422,6 +1423,41 @@ def _va_for_angle(self, angle):
             return 'baseline' if anchor_at_left else 'top'
         return 'top' if anchor_at_left else 'baseline'
 
+    def get_language(self):
+        """Return the language this Text is in."""
+        return self._language
+
+    def set_language(self, language):
+        """
+        Set the language of the text.
+
+        Parameters
+        ----------
+        language : str or None
```

Comment:
```suggestion
        language : str or list or tuple or None
```

Reply:
I was about the ask the same question looking at the tests! 

Marking this as resolved.